src/App.js

```diff
@@ -1,12 +1,52 @@
 import React from "react";
 import "./App.css";
+import { useState } from "react";
 
-function App () {
-    return (
-      <main>
-        <p>React State Lab</p>
-      </main>
-    );
+function App() {
+  const buttons = document.querySelectorAll("button");
+
+  function addToUserScore(addToScore) {
+    setNewScore(startingScore + addToScore);
+  }
+
+  function changeAddToScore(addToScore) {
```

Comment:
Don't have to pass `addToScore` as a param here, since it's in the state, you'll be able to access it without passing it as a param (the function is at the same scope level as the state values, so the code inside the function has access to those variables)